packages/line/index.d.ts

```diff
@@ -83,6 +83,29 @@ declare module '@nivo/line' {
     export type LineCustomLayer = (props: LineCustomLayerProps) => React.ReactNode
     export type Layer = LineLayerType | LineCustomLayer
 
+    export interface LineTooltipProps {
+        point: {
+            id: string
+            index: number
+            serieId: string | number
+            serieColor: string
+            x: number
+            y: number
+            color: string
+            borderColor: string
+            data: {
+                color: string
+                x: string | number
```

Comment:
x and y values can be numbers, strings or dates, so the type should be `number | string | Date`.

Reply:
Updated.